File: anacondacode/tensorflow_environment/old_code/计算方法/homework2_2.py

#幂函数，指数，阶乘
import math
import numpy as np
import matplotlib.pyplot as plt

N=6
#n=1:N
n=np.linspace(1,N)
x1=np.power(n,2)
x2=np.power(2,n)

x3=[]
for i in range(1,N+1):
    x3.append(math.factorial(i))


plt.plot(n,x1,'r')
plt.plot(n,x2,'k')
plt.plot(x3,'b')

File: matlab_code/oldcode/homework2_2.m
%幂函数，指数，阶乘
N=6;
n=1:N;
x1=n.^2;x2=2.^n;x3=[];
for i=1:N
    x3(i)=factorial(i);
end
plot(n,x1,'r');
hold on;plot(n,x2,'k');
hold on;plot(n,x3,'b');
legend('幂','指数','阶乘');